packages/api/src/external/carequality/patient.ts

```diff
@@ -20,9 +20,18 @@ import { toFHIR } from "../fhir/patient";
 import { makeIheGatewayAPI } from "./api";
 import { createPatientDiscoveryRequest } from "./create-pd-request";
 import { cqOrgsToXCPDGateways } from "./organization-conversion";
+import { MedicalDataSource } from "..";
+import { PatientDataCarequality } from "./patient-shared";
 
 dayjs.extend(duration);
 
+export function getCQData(
+  data: PatientExternalData | undefined
+): PatientDataCarequality | undefined {
+  if (!data) return undefined;
+  return data[MedicalDataSource.CAREQUALITY] as PatientDataCarequality; // TODO validate the type
+}
```

Comment:
nit: forgot the todo here? remove type assertion if possible

Reply:
Its similar to what we have in CW where we assert the type so we dont have to do it in multiple places